cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -24,11 +24,20 @@ message CassandraFrameworkConfiguration {
     optional int64 memMb = 6;
     optional int64 diskMb = 7;
     optional int64 healthCheckIntervalSeconds = 8;
+    optional int32 numberOfSeeds = 9;
+    optional int64 bootstrapGraceTimeSeconds = 10;
+    repeated PortMapping portMapping = 11;
+}
+
+message PortMapping {
+    required string name = 1;
+    required int32 port = 2;
 }
 
 message CassandraClusterState {
     repeated CassandraNode nodes = 1;
     repeated ExecutorMetadata executorMetadata = 2;
+    optional int64 lastServerLaunchTimestamp = 3;
```

Comment:
I don't know if we should be using time to determine when to launch servers rather than events and state.